src/cmd.ts

```diff
@@ -73,6 +73,10 @@ function createCommandWithSharedOptions(name, description) {
 			"Safe will only download torrents with perfect matches. Risky will allow for renames and more matches, but might cause false positives",
 			fallback(fileConfig.dataMode, "safe")
 		)
+		.option(
+			"-h, --hardlink-dir <dir>",
```

Comment:
```suggestion
			"--link-dirs <dirs...>",
```

maybe i'm getting ahead of myself here, but given that hardlinks don't work across mount points, i could see us needing at most one link dir per data dir, if each data dir is on a different volume. it doesn't feel great but it could be an implicit mapping where order matters, and then it'd be at least supported. but maybe that's a feature for later. 

also, we can always add short params later but taking them away is hard so let's keep it long for now

Reply:
+1 on long name, however initially I think supporting one dir is preferable. Getting into mapping is weird especially if you don't have an even number of source and dest dirs. I suppose there could be a validation that there is either one dest dir, or as many as there are sources, but imo that should be more formalized. Too easy for users to create errors even with saying there's an implicit hidden mapping and the dir order they provides matters.

This is one of those things that I think would be better suited in a dummy proof UI when that comes down the pipeline.

Reply:
yeah, agreed hard on the web ui. Trying to fit all the configuration data into a format that's specifiable on the command line sucks.

> initially I think supporting one dir is preferable

ok, sounds good.